src/components/content/catalog/services/details/ServiceProvider.tsx

```diff
@@ -209,46 +206,109 @@ function ServiceProvider({
-                                updateServiceHostingType={onChangeServiceHostingType}
-                            />
-                            <ServiceDetail serviceDetails={activeServiceDetail} />
-                            <ServicePolicies
-                                key={activeServiceDetail.serviceTemplateId}
-                                serviceDetails={activeServiceDetail}
-                                isViewDisabled={isViewDisabled}
-                            />
+                            {(activeServiceDetail.serviceTemplateRegistrationState === 'in-review' ||
+                                activeServiceDetail.serviceTemplateRegistrationState === 'approved') &&
+                            !activeServiceDetail.availableInCatalog ? (
+                                <Badge.Ribbon
+                                    placement={'start'}
+                                    text={<div className={catalogStyles.serviceTemplateState}>Review In-Progress</div>}
+                                    color='#e67300'
+                                >
+                                    <ServiceTemplateDetails
+                                        isViewDisabled={isViewDisabled}
+                                        serviceDetails={activeServiceDetail}
+                                        groupServiceTemplatesByCsp={groupServiceTemplatesByCsp}
+                                        serviceCspInQuery={serviceCspInQuery}
+                                        serviceHostingTypeInQuery={serviceHostingTypeInQuery}
+                                        onChangeServiceHostingType={onChangeServiceHostingType}
+                                    />
+                                </Badge.Ribbon>
+                            ) : activeServiceDetail.serviceTemplateRegistrationState === 'approved' &&
+                              activeServiceDetail.availableInCatalog ? (
+                                <Badge.Ribbon
+                                    placement={'start'}
+                                    text={<div>Available In Catalog</div>}
+                                    color='#87d068'
+                                >
+                                    <ServiceTemplateDetails
+                                        isViewDisabled={isViewDisabled}
+                                        serviceDetails={activeServiceDetail}
+                                        groupServiceTemplatesByCsp={groupServiceTemplatesByCsp}
+                                        serviceCspInQuery={serviceCspInQuery}
+                                        serviceHostingTypeInQuery={serviceHostingTypeInQuery}
+                                        onChangeServiceHostingType={onChangeServiceHostingType}
+                                    />
+                                </Badge.Ribbon>
+                            ) : activeServiceDetail.serviceTemplateRegistrationState === 'approved' &&
```

Comment:
As discussed, in case of a service available in catalog but when review of an update pending for review, the badge is wrong. 
We will need both this information to be displayed.
Service Available in catalog 
Updated template review in progress.